File: loiane-training-java/basico/Exer Aula 18_19/Ex33.java
public class Ex33 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 8, 9, 10 };

        for (int i = 1; i <= a.length; i++) {
            var prime = true;
            for (int j = 2; j < i; j++) {
                if (a[i] % j == 0) {
                    prime = false;
                    break;
                }

            }
            if (prime) {
                System.out.println(i + "\tPrimo");
            } else {
                System.out.println(i + "\tNao Primo");
            }
        }
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex33.cs
using System;

class Ex33
{
    static void Main(string[] args)
    {
        int[] vetorA = new int[10] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };

        for (int i = 0; i < vetorA.Length; i++)
        {
            bool isPrime = true;

            //Verifica se e primo
            if (vetorA[i] < 1)
            {
                Console.WriteLine("Primo");
                isPrime = false;
            }
            else
            {
                for (int j = 2; j < vetorA[i]; j++)
                {
                    if (vetorA[i] % j == 0)
                    {
                        isPrime = false;
                    }
                }
            }

            if (isPrime)
            {
                Console.WriteLine(vetorA[i] + ": Primo");
            }
            else
            {
                Console.WriteLine(vetorA[i] + ": Nao primo");
            }
        }
    }
}